app/containers/EmojiPicker/categories.ts

```diff
@@ -1,43 +1,43 @@
 const list = ['frequentlyUsed', 'custom', 'people', 'nature', 'food', 'activity', 'travel', 'objects', 'symbols', 'flags'];
```

Comment:
```suggestion
const list = ['clock', 'rocket', 'emoji' ...];
```
Isn't it easier to correct the names here and make a map from this list?

Reply:
Using category name ('custom', 'people' ...) instead of icon name ('rocker', 'emoji' ...) makes more sense IMO. Also we'll have to change the keys in `./emoji.ts` to these icon names.